clang/lib/Sema/SemaTemplateInstantiateDecl.cpp

```diff
@@ -6481,6 +6481,33 @@ void Sema::PerformPendingInstantiations(bool LocalOnly) {
     PendingInstantiations.swap(delayedPCHInstantiations);
 }
 
+// Instantiate all referenced specializations of the given function template
+// definition. This make sure that function template that are defined after the
+// point of instantiation of their used can be evaluated after they are defined.
+// see CWG2497.
+void Sema::InstantiateFunctionTemplateSpecializations(
```

Comment:
This name doesn't describe what the function does very well. Maybe something more like `PerformPendingInstantiationsOfConstexprFunction` would be clearer?